_posts/2023-08-23-dashboards-usability-snapshot.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "OpenSearch Dashboards: A snapshot of the current usability"
```

Comment:
Would "OpenSearch Dashboards: A usability snapshot" be better?

Reply:
Yes